### PR TITLE
Dutch translation

### PR DESCRIPTION
**Pull request by Bond-009**

### CODE REVIEW

File: app/src/main/res/values-nl/strings.xml [hunk 1]
```diff
@@ -0,0 +1,73 @@
+<resources>
+    <string name="do_install_cuberite">Installeer Cuberite</string>
+    <string name="do_start_cuberite">Start Server</string>
+    <string name="do_stop_cuberite">Stop Server</string>
+    <string name="do_kill_cuberite">Forceer stop Server (Als gewoon stoppen niet werkt)</string>
+    <string name="do_execute_line">Voer uit!</string>
+    <string name="username">Gebruikersnaam</string>
+    <string name="password">Paswoord</string>
```
Comment (NiLSPACE) [inline app/src/main/res/values-nl/strings.xml:8]: `Wachtwoord`

File: app/src/main/res/values-nl/strings.xml [hunk 2]
```diff
@@ -0,0 +1,73 @@
...
+
+    <string name="settings_install_heading">Installatie opties</string>
+    <string name="settings_install_update">Update Cuberite</string>
+    <string name="settings_install_update_explanation">Er is voorlopig nog geen automatische updater, gebruike deze sectie van tijd tot tijd om uw Cuberite installatie te updaten. Opmerking: als u een server.zip update uitvoert is er een kans dat uw server instellingen verloren gaan.</string>
+    <string name="settings_install_update_binary">Download en installeer de Cuberite binary</string>
+    <string name="settings_install_update_server">Download en installeer server.zip</string>
+    <string name="settings_install_manually">Installeer zip bestanden manueel</string>
+    <string name="settings_install_manually_explanation">In plaats van Cuberite te downloaden kan u ervoor kiezen om uw eigen gecompliceerde binaries te gebruiken. Gebruik dit enkel als u weet wat u doet.</string>
```
Comment (NiLSPACE) [inline app/src/main/res/values-nl/strings.xml:42]: `gecompliceerde` -> `gecompileerde`

File: app/src/main/res/values-nl/strings.xml [hunk 3]
```diff
@@ -0,0 +1,73 @@
...
+    <string name="settings_install_no_sha_button">Download and installeer Cuberite</string>
+
+    <string name="settings_webadmin_heading">Webadmin instellingen</string>
+    <string name="settings_webadmin_explanation">Voor diepgaande instellingen gebruikt u best de ingebouwde Webadmin. Gebruik deze optie om een gebruikersnaam en paswoord voor de Webadmin in te stellen.</string>
+    <string name="settings_webadmin_login">Verander gebruikersnaam en paswoord</string>
+    <string name="settings_webadmin_not_installed_title">Cuberite is nog niet geïnstalleerd</string>
+    <string name="settings_webadmin_not_installed">Voor de gebruikersnaam en paswoord aan te passen moet Cuberite geïnstalleerd zijn. U kan kiezen om dit nu te doen of om dit dialoog venster te sluiten.</string>
+    <string name="settings_webadmin_error">Er ging iets fout tijdens het aanpassen van de Webadmin configuratie</string>
+    <string name="settings_webadmin_success">Success! De nieuwe gebruikersnaam en paswoord zijn geldig nadat u Cuberite herstart</string>
```
Comment (NiLSPACE) [inline app/src/main/res/values-nl/strings.xml:56]: `paswoord` -> `wachtwoord`

File: app/src/main/res/values-nl/strings.xml [hunk 4]
```diff
@@ -0,0 +1,73 @@
+<resources>
+    <string name="do_install_cuberite">Installeer Cuberite</string>
+    <string name="do_start_cuberite">Start Server</string>
+    <string name="do_stop_cuberite">Stop Server</string>
+    <string name="do_kill_cuberite">Forceer stop Server (Als gewoon stoppen niet werkt)</string>
+    <string name="do_execute_line">Voer uit!</string>
+    <string name="username">Gebruikersnaam</string>
+    <string name="password">Paswoord</string>
+    <string name="enabled">Ingeschakeld</string>
+    <string name="disabled">Uitgeschakeld</string>
```
Comment (NiLSPACE) [inline app/src/main/res/values-nl/strings.xml:10]: `enabled` and `disabled` are both lowercase in the English version. There could be a reason for that.

File: app/src/main/res/values-nl/strings.xml [hunk 5]
```diff
@@ -0,0 +1,73 @@
...
+    <string name="status_missing_filemanager">Geen bestandsbeheerder gevonden: installeer er a.u.b een</string>
+
+    <string name="notification_cuberite_running">Cuberite draait!</string>
+
+    <string name="service_description">Deze service draait Cuberite and zorgt ervoor dat deze niet wordt gestopt door het systeem.</string>
+    <string name="installservice_description">Deze service download, unzipt en installeert Cuberite.</string>
+
+    <string name="inputLine_hint">Voer commando's in…</string>
+
+    <string name="message_externalstorage_permission">Cuberite bewaart de werelden standaard op externe opslagmedia, die lees- en schrijfbaar zijn door iedere app. Hiervoor moet u Cuberite toestaan om te schrijven naar het externe medium. Als u de nodige toestemming niet wilt geven, selecteer de \"vraag niet opnieuw\" optie en Cuberite zal de app zijn privé opslag gebruiken.</string>
```
Comment (NiLSPACE) [inline app/src/main/res/values-nl/strings.xml:29]: The comma after `opslagmedia` shouldn't be there.

**Comment by NiLSPACE** [comment]
This PR was merged a little too soon ;)

File: app/src/main/res/values-nl/strings.xml (hunk 1 above)
Comment (Bond-009) [reply app/src/main/res/values-nl/strings.xml:8]: Paswoord is a correct translation

**Comment by Bond-009** [comment]
@NiLSPACE #8

File: app/src/main/res/values-nl/strings.xml (hunk 1 above)
Comment (madmaxoft) [reply app/src/main/res/values-nl/strings.xml:8]: Even the Dutch can't agree on the translation :)

Comment (Bond-009) [reply app/src/main/res/values-nl/strings.xml:8]: Belgium vs the Netherlands :)

Comment (NiLSPACE) [reply app/src/main/res/values-nl/strings.xml:8]: Is `paswoord` more common in Belgium? In The Netherlands we always use `wachtwoord`.

Comment (Bond-009) [reply app/src/main/res/values-nl/strings.xml:8]: I personally use both, but I changed it in my new pr to `wachtwoord`.

File: app/src/main/res/values-nl/strings.xml (hunk 4 above)
Comment (ItsShadowCone) [reply app/src/main/res/values-nl/strings.xml:10]: Late response, wasn't on the Internet. Reason is "Authentication is now en/disabled" thanks for pointing it out

**Comment by ItsShadowCone** [comment]
I wasn't aware those pr's get reviewed ;) usually I would review it in here but it's not my language :)

**Pull request merged**